operator/cmd/mesh/manifest-validate.go

```diff
@@ -38,21 +44,25 @@ var (
 	boolValues = []bool{true, false}
 
 	// Ref: https://kubernetes.io/docs/concepts/configuration/overview/#container-images
-	imagePullPolicy = []string{"Always", "IfNotPresent", "Never"}
+	imagePullPolicy = []string{
+		string(kubeCore.PullAlways),
+		string(kubeCore.PullNever),
+		string(kubeCore.PullIfNotPresent),
+	}
 
-	// Keep this list updated as per following
+	// List of available profiles from operator/data/profiles
 	// https://preliminary.istio.io/docs/setup/additional-setup/config-profiles/
-	profile = []string{"default", "demo", "empty", "minimal", "preview", "remote", "separate"}
+	profile = getProfiles()
 
 	inboundClusterStatName = []string{"SERVICE", "SERVICE_FQDN", "SERVICE_PORT", "SERVICE_PORT_NAME"}
```

Comment:
I will have to create enum in `istio/api` as MeshConfig related proto are defined over there. Then only I will be able to update these values. I will create a separate PR for the same.

Reply:
Yes, that's a good idea. We should have enums for all fields of this type and I think some are still string type unfortunately. Thanks.